src/System.Runtime.Numerics/tests/BigInteger/cast_from.cs

```diff
@@ -564,6 +575,12 @@ public static void RunDoubleExplicitCastFromBigIntegerTests()
         }
 
         [Fact]
```

Comment:
I think this one needs to be OuterLoop as well. I think you may have just forgotten the attribute as you did put it on the other separated test method below. This one took 32 seconds out of a total 35 seconds for the whole test run on my machine.